simphony/cuds/abc_modeling_engine.py

```diff
@@ -37,16 +37,9 @@ def add_lattice(self, lattice):
 
         Parameters
         ----------
-        lattice: ABCLattice
+        lattice : ABCLattice
             lattice to be added.
 
-        Returns
```

Comment:
Is the "Returns" section removed from the add_XXX methods because one can already read it in get_XXX()?